src/pages/IDO/index.tsx

```diff
@@ -0,0 +1,172 @@
+import { darken } from 'polished'
+import { CardSection, DataCard, CardNoise, CardBGImage } from '../../components/earn/styled'
+
+import yaygamesIcon from '../../assets/images/yaygames-icon.png'
+import boofinanceIcon from '../../assets/images/boofinance-icon.jpeg'
+
+
+import React from 'react'
+import { AutoColumn } from '../../components/Column'
+import styled from 'styled-components'
+import { TYPE, ExternalLink } from '../../theme'
+
+import { RowBetween } from '../../components/Row'
+
+import { useTranslation } from 'react-i18next'
+
+const PageWrapper = styled(AutoColumn)``
+
+const TopSection = styled(AutoColumn)`
+  max-width: 640px;
+  width: 100%;
+`
+
+const VoteCard = styled(DataCard)`
```

Comment:
Let's call this something more appropriate for the page

Reply:
Renamed to IDOCard